android/src/main/java/com/cloudwebrtc/webrtc/MethodCallHandlerImpl.java

```diff
@@ -980,14 +988,14 @@ public String peerConnectionInit(ConstraintsMap configuration, ConstraintsMap co
     if (mPeerConnectionObservers.size() == 0) {
       if (Build.VERSION.SDK_INT < Build.VERSION_CODES.S
               || context.getApplicationInfo().targetSdkVersion < Build.VERSION_CODES.S) {
-        audioManager.onAudioManagerRequested(true);
+        //audioSwitchManager.start();
```

Comment:
Look strange that this if execute nothing.  I didn't know if we need to start the manager or not.  But having a if that does nothing seems useless

Reply:
I rechecked the code, this piece should be able to be removed,
audioSwitchManager has already called start when it is created, and start is only used to monitor device changes, so it will not take over the audio route

Reply:
Is this Bluetooth permission necessary?

```dart
...
        ArrayList<String> permissions = new ArrayList<>();
        permissions.add(Manifest.permission.BLUETOOTH_CONNECT);
        requestPermissions(
                permissions,
                (args) -> {
                  //audioSwitchManager.start();
                },
                (args) -> {
                });
      }
    }
...
```

I'm getting requested access for nearby devices but from my understanding, this shouldn't be required.

Can it be removed? If not, is there a way to prevent this line from being called? 

Thanks in advance!